### PR TITLE
goodbye axios

### PR DESCRIPTION
**Pull request by hkubus**

### CODE REVIEW

File: src/commands/skywars.ts [hunk 1]
```diff
@@ -23,10 +25,12 @@ export async function run(
             ),
             ephemeral: true
         });
-
-    const { data }: { data: HypixelType } = await axios(
+    // @ts-ignore
+    const { data }: { data: HypixelType } = await fetch(
```
Comment (SuperchupuDev) [inline src/commands/skywars.ts]: change this to
```ts
const data = (await fetch(
        `https://api.hypixel.net/player?uuid=${uuid.data.id}&key=${client.config.hypixel}`
    ).then(res => res.json())) as HypixelType;
```
just like you did with bedwars and remove that `@ts-ignore`

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]

File: src/commands/skywars.ts (hunk 1 above)
Comment (hkubus) [reply src/commands/skywars.ts]: Thats

Comment (hkubus) [reply src/commands/skywars.ts]: The ts ignore is needed cause it does type {} funni missing

Comment (hkubus) [reply src/commands/skywars.ts]: oh

**Review by SuperchupuDev — APPROVED** [review]

**Pull request merged**